### DISCUSSION TITLE
New slicer option: Q slicer

### DISCUSSION THREAD

**Discussion by astellhorn** [SasView/sasview#2198] [General]
Add new slicer option: "Q slicer"
- need sector box and annular box --> convolute them
- get a Q slicer box looking like in the attached image
- optimum: similar as requested in #2077 it should be made possible to add several boxes as shown in the image below.
(i) either with d_phi/ dQ completely user defined for EACH BOX
(ii) or with d_phi/ dQ mirrored at Qx=0 / Qy=0
(iii) or with a defined rotation in phi from box to box (e.g. Box 1 at phi1, Box 2 at phi1+45deg...)

![Q slicer](https://user-images.githubusercontent.com/93910032/173855010-22968715-6cd2-4d0f-90e0-cce47c566fbd.jpg)

**Comment by butlerpd** [comment]
The first part of this has been done and is in 6.0. Only the last part (providing multiple slicer widgets on a single plot) remains and is actively being discussed though not yet worked on.

**Comment by jellybean2004** [comment]
This is now completed with #3832.
